context.go

```diff
@@ -109,31 +110,42 @@ func CreateContext(masterKey, masterSalt []byte, profile ProtectionProfile, opts
 
 // https://tools.ietf.org/html/rfc3550#appendix-A.1
 func (s *srtpSSRCState) nextRolloverCount(sequenceNumber uint16) (uint32, func()) {
-	roc := s.rolloverCounter
-
-	switch {
-	case !s.rolloverHasProcessed:
-	case sequenceNumber == 0: // We exactly hit the rollover count
-		// Only update rolloverCounter if lastSequenceNumber is greater then maxROCDisorder
-		// otherwise we already incremented for disorder
-		if s.lastSequenceNumber > maxROCDisorder {
-			roc++
+	seq := int32(sequenceNumber)
+	localRoc := uint32(s.index >> 16)
+	localSeq := int32(s.index & 0xffff)
+
+	guessRoc := localRoc
+	var difference int32 = 0
+
+	if s.rolloverHasProcessed {
```

Comment:
https://github.com/cisco/libsrtp/blob/bd0f27ec0e299ad101a396dde3f7c90d48efc8fc/crypto/replay/rdbx.c#L309-L325
I think above condition (`s.index > seqNumMedian`) should be checked to avoid wrong guess after initialization and ROC overflow.

Reply:
Yes, this judgment needs to be added.

Reply:
@at-wat  Sorry, I forgot to run the check script. This is my first PR... T_T